cms/djangoapps/contentstore/views/course.py

```diff
@@ -1422,6 +1453,18 @@ def group_configurations_detail_handler(request, course_key_string, group_config
             return JsonResponse(status=204)
 
 
+def has_course_enabled_content_experiments(course):
+    """
+    Returns True if Studio should show the "Experiment Group
```

Comment:
I have to admit that I hate the name of this method now. I've been trying to keep it to myself, but I can't.

How about are_content_experiments_enabled?